src/core.js

```diff
@@ -1680,11 +1680,17 @@ define([
                 isCopyable: !multiselect && mug.options.isCopyable
             }));
         $baseToolbar.find('.fd-button-remove').click(function () {
-            var mugs = _this.getCurrentlySelectedMug(true);
-            if (mugs.length > 1 || (mugs.length && form.getChildren(mugs[0]).length)) {
+            var mugs = _this.getCurrentlySelectedMug(true),
+                errorMessage;
+            if (mugs.length > 1) {
+                errorMessage = "You are about to delete multiple questions. This cannot be undone";
+            } else if (mugs.length && form.getChildren(mugs[0]).length) {
+                errorMessage = "You are about to delete a question with child nodes. This cannot be undone";
```

Comment:
Ugh, writing succinct error messages that are actually helpful is so hard. I do not like this confirmation dialog at all. It interrupts the workflow to ask a silly question that is unexpected and annoying. This is why I want to implement undo.

Another thing to consider is that people are often speed-reading, so they may only read the first few words, and maybe the last few words. Starting with "You are about to..." forces the reader to slow down and read more carefully. I'm standing by my suggestion: the succinctness of "Deleting multiple questions..." is worth the minor inaccuracy for select questions. An alternative that would avoid the inaccuracy would be to delete select questions without prompting, but that would make the implementation more complicated.

I'm really curious about the original suggestion that prompted this, but I have not been able to find it. Jon, can you point me to it?